test/pico_sha256_test/CMakeLists.txt

```
@@ -1,4 +1,5 @@
-if (NOT TARGET hardware_sha256)
+if (NOT TARGET pico_sha256)
+    message("Skipping hardware_sha256_test as pico_sha256 is unavailable on this platform")
```

Test name is actually `pico_sha256_test`, so maybe?
```suggestion
    message("Skipping pico_sha256_test as pico_sha256 is unavailable on this platform")
```